Content.Client/Doors/DoorSystem.Airlock.cs

```diff
@@ -82,23 +79,31 @@ private void OnAppearanceChange(EntityUid uid, AirlockComponent comp, ref Appear
         var emergencyLightsVisible = false;
         var unlitVisible = false;
 
-        if (!_appearanceSystem.TryGetData<DoorState>(uid, DoorVisuals.State, out var state, args.Component))
+        if (!Appearance.TryGetData<DoorState>(uid, DoorVisuals.State, out var state, args.Component))
             state = DoorState.Closed;
 
-        if (_appearanceSystem.TryGetData<bool>(uid, PowerDeviceVisuals.Powered, out var powered, args.Component)
+        if (Appearance.TryGetData<bool>(uid, PowerDeviceVisuals.Powered, out var powered, args.Component)
             && powered)
         {
-            boltedVisible = _appearanceSystem.TryGetData<bool>(uid, DoorVisuals.BoltLights, out var lights, args.Component)
-                            && lights && (state == DoorState.Closed || state == DoorState.Welded);
-
-            emergencyLightsVisible = _appearanceSystem.TryGetData<bool>(uid, DoorVisuals.EmergencyLights, out var eaLights, args.Component) && eaLights;
+            boltedVisible =
+                Appearance.TryGetData<bool>(uid, DoorVisuals.BoltLights, out var lights, args.Component)
+                && lights && (state == DoorState.Closed || state == DoorState.Welded);
+
+            emergencyLightsVisible =
+                Appearance.TryGetData<bool>(uid,
+                    DoorVisuals.EmergencyLights,
+                    out var eaLights,
+                    args.Component) && eaLights;
             unlitVisible =
-                    (state == DoorState.Closing
-                ||  state == DoorState.Opening
-                ||  state == DoorState.Denying
-                || (state == DoorState.Open && comp.OpenUnlitVisible)
-                || (_appearanceSystem.TryGetData<bool>(uid, DoorVisuals.ClosedLights, out var closedLights, args.Component) && closedLights))
-                    && !boltedVisible && !emergencyLightsVisible;
+                (state == DoorState.Closing
+                 || state == DoorState.Opening
+                 || state == DoorState.Denying
+                 || (state == DoorState.Open && comp.OpenUnlitVisible)
+                 || (Appearance.TryGetData<bool>(uid,
+                     DoorVisuals.ClosedLights,
+                     out var closedLights,
+                     args.Component) && closedLights))
+                && !boltedVisible && !emergencyLightsVisible;
```

Comment:
```suggestion
                (state == DoorState.Closing ||
                state == DoorState.Opening ||
                state == DoorState.Denying ||
                (state == DoorState.Open && comp.OpenUnlitVisible) ||
                (Appearance.TryGetData<bool>(uid,
                     DoorVisuals.ClosedLights,
                     out var closedLights,
                     args.Component) && closedLights)) &&
                 !boltedVisible && !emergencyLightsVisible;
```

double check the indentation because I certainly have no idea how it's supposed to work.

Reply:
These should be using pattern matching. Tempted resolve as out of scope but will leave open

Reply:
No problem resolving all of them without changes if the whole construct is blasted away 🫡